virt/support/virt-support-overview.adoc

```diff
@@ -35,6 +35,9 @@ The {product-title} web console displays resource usage, alerts, events, and tre
 
 |*VirtualMachines* -> *VirtualMachine* -> *VirtualMachine details* -> xref:../../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-events[*Events* tab]
 |List of VM events
+
+|*VirtualMachines* -> *VirtualMachine* -> *VirtualMachine details* -> xref:../../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-diagnostics_virt-web-console-overview[*Diagnostics tab*]
```

Comment:
You need to remove "_virt-web-console-overview". That's the "context" attribute. Since I removed it from the web console overview, this xref is broken.